internal/generate/generate_helper.go

```diff
@@ -45,6 +45,19 @@ func GenerateArtifactsHelper(t *templator.Templator, cfg *config.Commit0Config,
 		kubernetes.Generate(t, cfg, &wg, pathPrefix)
 	}
 
+	log.Println(aurora.Cyan(emoji.Sprintf("Generating Terraform")))
+	terraform.Generate(t, cfg, &wg, pathPrefix)
+	if cfg.Infrastructure.AWS.Cognito.Deploy {
+
```

Comment:
This is a temporary solution. We'll need to figure out a system to the scenarios where one part of the code generation requires some output from another step.

Reply:
Hm yeah, that's gonna get complicated..